A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A320_Neo_MFD.js

```diff
@@ -616,11 +616,9 @@ class A320_Neo_MFD_Map extends MapInstrumentElement {
         const url = document.getElementsByTagName("a320-neo-mfd-element")[0].getAttribute("url");
         const screenIndex = parseInt(url.substring(url.length - 1));
         this.potIndex = screenIndex == 1 ? 89 : 91;
-        this.updateThrottler = new UpdateThrottler(screenIndex === 1 ? 300 : 600);
-        this.mcduWaypointCheckThrottler = new UpdateThrottler(50);
+        this.updateThrottler = new UpdateThrottler(screenIndex === 1 ? 200 : 400);
+        this.mdcuWaypointCheckThrottler = new UpdateThrottler(50);
```

Comment:
why M D C U instead of M C D U ?

Reply:
The typo was fixed in the PR that is reverted here, so the revert is undoing the fix.